public/index.html

```diff
@@ -45,5 +45,38 @@
       onreadystatechange="if (this.readyState === 'complete') this.onload()"
     ></script>
     <div id="root"></div>
+    <script>
+     var {pathname} = window.location;
+
+     var manifest = {
+         "short_name": "organice",
+         "name": "organice",
+         "icons": [
+             {
+                 "src": "organice-512x512.png",
+                 "sizes": "512x512",
+                 "type": "image/png"
+             },
+             {
+                 "src": "organice-192x192.png",
+                 "sizes": "192x192",
+                 "type": "image/png"
+             },
+             {
+                 "src": "favicon.ico",
+                 "sizes": "64x64 32x32 24x24 16x16",
+                 "type": "image/x-icon"
+             }
+         ],
+         "start_url": `${pathname || "/index.html"}`,
```

Comment:
No need for the template literal here?

```suggestion
         "start_url": pathname || "/index.html",
```

Reply:
:+1: